lib/steps.js

```diff
@@ -777,6 +777,12 @@ When(/I set the name of ?(?:the var(?:iable)?)? "([^"]+)" to "([^"]+)"/, async (
   ];
 
   let name_parts = name_str.split(' ');
+
+  if(name_parts.length > 4) {
+    await scope.addToReport(scope, { type: `warning`, value: `There were too many inputs in the name field` });
+    name_parts = name_parts.slice(0, 4);
+  }
+
```

Comment:
Unsure of the warning language and style, let me know what you prefer there.

Reply:
This was great! It used our format to great effect. I made a "suggestion" for the warning language that I think gives more information. You can give me feedback on it or you can just commit it, whichever you prefer.

Reply:
hmm either works for me. Does it need to consider how people can make mistakes and autofill? Or am I REALLY over thinking it?

Reply:
I had never considered autofill. Dang. We'll have to think about that, but maybe as a separate exploratory issue.

Regarding other mistakes, [anything should be permissible for names](https://www.kalzumeus.com/2010/06/17/falsehoods-programmers-believe-about-names/), so it can't give wrong input for the text fields. If you mean the dropdown, that would be a developer mistake and they'd need to fix that.

To explain more and [increase the bus factor](https://developerexperience.io/practices/bus-factor), I figure that a developer is much more likely to add an extra name in the middle than they are to get the wrong suffix (final string). We sort of see it in your test example - the last word in the string is "III".

If we do this, we might tweak the test - instead of having the last two strings be suffixes, only the last one should be a valid suffix.

Reply:
Autofill: Do we press "enter" after filling in a field? If not, we should be good.

Reply:
We don't! We just get into the next field from what I can tell. Unless puppeteer is doing something weird and pressing `enter` somewhere, we don't do it ourselves, so we should be good. Thanks for bringing it up, it was a great thing to check on!